tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
   lint
-  py{36,37,38}-{base,full}
+  py-{base,full}
```

Comment:
question: do we not need the `[gh-actions]` section, per the docs? https://github.com/ymyzk/tox-gh-actions#tox-gh-actions-configuration. The attrs config also includes it: https://github.com/python-attrs/attrs/blob/72f94a3b03d5c2f8211f499c1c3fc72413d3fd4e/tox.ini#L11

Reply:
yeah. so, I don't really know tox much, but as far as I can understand, doesn't using something like github actions defy the purpose of tox? gh action already has the python matrix, so seems like they fulfill similar tasks...

Reply:
tox is also useful for local development. I use it for `tox -e lint` as well as reproducing CI failures. Removing Python version mgmt outside of tox does lessen the benefit, though.

Reply:
Just curious--do you use `tox-pyenv` or something similar? ie: how do you have multiple clean Python installs locally to test locally that's close to CI? (or is that the wrong strategy)